mmdet/ops/roi_pool/src/roi_pool_kernel.cu

```diff
@@ -14,7 +14,7 @@ inline int GET_BLOCKS(const int N) {
   return min(optimal_block_num, max_block_num);
 }
 
-template <typename scalar_t>
+template<typename scalar_t>
```

Comment:
There are still some formatting problems. For example, this line should be the same as https://github.com/open-mmlab/mmdetection/blob/d132051dd8cce83c14677d565705d5049150d986/mmdet/ops/roi_align/src/roi_align_kernel.cu#L17